src/Symfony/Component/ExpressionLanguage/Node/GetAttrNode.php

```diff
@@ -70,6 +70,10 @@ public function evaluate($functions, $values)
 
                 $property = $this->nodes['attribute']->attributes['value'];
 
+                if (!property_exists($obj, $property) && !method_exists($obj, '__get')) {
+                    throw new \RuntimeException('Unable to get a non-supported property.');
```

Comment:
The error message should be more descriptive and give information about the property name and probably the class name as well.